DigitalLearningSolutions.Web/Styles/trackingSystem/reports.scss

```diff
@@ -93,3 +93,15 @@ $light-blue: #8EBAFF;
     transform: translate(-100%) rotate(-60deg);
   }
 }
+
+.eval-summary-row__key {

```

Comment:
Do we not want to use width defaults by NHS?

Reply:
The default gives more width to the value and less to the key, but here we need more space for the key (question) and less for the value (percentage).

Reply:
That makes sense :)